src/api/procedures/types.ts

```diff
@@ -488,7 +488,7 @@ export type AddInstructionWithVenueIdParams = AddInstructionParams & {
   venueId: BigNumber;
 };
 
-export interface AffirmInstructionParams {
+export interface InstructionIdParams {
```

Comment:
I think this is a more accurate name for it. In the affirm method the intent is still clear due to the method name.